pkg/starter/starter.go

```diff
@@ -192,7 +192,7 @@ func (s *Starter) processJob(ctx context.Context, job datastore.Job) error {
 		IPAddress:      ipAddress,
 		TargetID:       job.TargetID,
 		CloudID:        cloudID,
-		ResourceType:   target.ResourceType,
+		ResourceType:   resourceType,
```

Comment:
`resoruceType` response is **optional** for backward compatibility.

You need to check that `resourceType` is nil. Please set `target.ResourceType` if nil.

Reply:
If `resoruceType` response is zero (= unknown), then reference the value of `target.ResourceType` (I think my way is not good).
https://github.com/whywaita/myshoes/pull/169/commits/20ce141cf376298b5a9aec23a2f99186115f725b#diff-3a0c64b483dd2fed243139d0725e8be15f12e642cd167cd9ce85923e63d138c5R170-R172

Also, there was an omission in the fix for shoes.go, so I fixed it.
https://github.com/whywaita/myshoes/pull/169/commits/20ce141cf376298b5a9aec23a2f99186115f725b#diff-e1765ca0db77f2dddd1d22c7afe51ed99e53823c487b61eefd75950772097778R82-R94